src/MongoDB.Driver.Legacy/CommandResults/MapReduceResult.cs

```diff
@@ -116,6 +119,7 @@ public IEnumerable<BsonDocument> InlineResults
         /// <summary>
         /// Gets the input count.
         /// </summary>
+        [Obsolete("InputCount is ignored by server versions 4.3.1 and newer.")]
```

Comment:
Slight rewording requested:

     [Obsolete("InputCount is not available on server versions 4.3.1 and newer.")]

Reply:
done